samples/esb/esb_ptx/README.rst

```diff
@@ -52,11 +59,16 @@ LED 1-4:
    The first four packets turn on LED 1, 2, 3, and 4.
    The next four packets turn them off again in the same order.
 
+Configuration
+*************
+
+|config|
+
 Building and running
 ********************
 
-The Transmitter sample can be found under :file:`samples/esb/ptx` in the |NCS| folder structure.
-The Receiver sample can be found under :file:`samples/esb/prx` in the |NCS| folder structure.
+The Transmitter sample can be found under :file:`samples/esb/esb_ptx` in the |NCS| folder structure.
```

Comment:
To my taste, I would rather say "is located" ...

Reply:
"Can be found" is how we have it in the building and running includes.

Reply:
Unfortunately so.

Reply:
For these lines, we have a replace in the sample template:

.. |sample path| replace:: :file:`samples/XXX`

We should use that, and we should not provide path to two samples within one sample.